how-to-use-azureml/azure-databricks/README.md

```diff
@@ -1,33 +1,73 @@
 
 **Databricks as a Compute Target from AML Pipelines**
-You can use Azure Databricks as a compute target from [Azure Machine Learning Pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/service/concept-ml-pipelines). Take a look at this notebook for details: [aml-pipelines-use-databricks-as-compute-target.ipynb](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/azure-databricks/databricks-as-remote-compute-target/aml-pipelines-use-databricks-as-compute-target.ipynb). 
+You can use Azure Databricks as a compute target from [Azure Machine Learning Pipelines](https://docs.microsoft.com/en-us/azure/machine-learning/service/concept-ml-pipelines). Take a look at this notebook for details: [aml-pipelines-use-databricks-as-compute-target.ipynb](https://github.com/Azure/MachineLearningNotebooks/tree/master/how-to-use-azureml/azure-databricks/databricks-as-remote-compute-target/aml-pipelines-use-databricks-as-compute-target.ipynb).
+
+# Linked Azure Databricks and Azure ML Workspaces (Preview)
+Customers can now link Azure Databricks and AzureML Workspaces to better enable MLOps scenarios by [managing their tracking data in a single place when using the MLflow client](https://mlflow.org/docs/latest/tracking.html#mlflow-tracking) - the Azure ML workspace.
+
+## Linking the Workspaces (Admin operation)
+
+1. The Azure Databricks Azure portal blade now includes a new button to link an Azure ML workspace
+![New ADB Portal Link button](./img/adb-link-button.png)
+2. Both a new or existing Azure ML Workspace can be linked in the resulting prompt. Follow any instructions to set up the Azure ML workspace.
+![Link Prompt](./img/link-prompt.png)
+3. After a successful link operation, you should see the Azure Databricks overview reflect the linked status
+![Linked Successfully](./img/adb-successful-link.png)
+
+## Configure MLflow to send data to Azure ML (All roles)
+
+1. Add azureml-mlflow as a library to any notebook or cluster that should send data to Azure ML. You can do this via
+    1. [DBUtils](https://docs.azuredatabricks.net/user-guide/dev-tools/dbutils.html#dbutils-library)
+        ```
+        dbutils.library.installPyPI("azureml-mlflow")
+        dbutils.library.restartPython()  # Removes Python state
+        ```
+    2. [Cluster Libraries](https://docs.azuredatabricks.net/user-guide/libraries.html#install-a-library-on-a-cluster)
+    ![Cluster Library](./img/cluster-library.png)
+2. [Set the MLflow tracking URI](https://mlflow.org/docs/latest/tracking.html#where-runs-are-recorded) to the following scheme:
+    ```
+    adbazureml://${azuremlRegion}.experiments.azureml.net/history/v1.0/subscriptions/${azuremlSubscriptionId}/resourceGroups/${azuremlResourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/${azuremlWorkspaceName}
+    ```
+    1. You can automatically configure this on your clusters using this helper script
```

Comment:
Is the init script valid only for this notebook or does it apply to any notebook created in this workspace? If i use the init script, do i still need to use a tracking URI?

Reply:
This is a cluster scoped init script so it applies to all notebooks created from that cluster you uploaded the init script to.
If you use that init script you do not need to manually set the tracking uri in the notebook.

Reply:
@balapv Added some more wording to address your questions - does that help?

Reply:
Looks good. Thanks